data/civ2civ3/README.civ2civ3

```diff
@@ -348,34 +374,40 @@ EXAMPLE: Riflemen in a size 9 city, with walls, on hills,
 
 GOVERNMENTS:
 ------------
-                  Anarch   Tribal   Despot   Monarc   Fundam   Republ   Democr   Federat  Commun   
-Tax/Lux/Sci Rate  0        60       60       70       70       80       90       90       80       
-Output per tile # -1(if>2) -1(if>2) -1(if>2) 0        0        +1Trade  +1Trade  0        0        
-Celebration Bonus (no -1)  (no -1)  (no -1) (+1Trade) 0       (+1Trade)(+1Trade)(+1Trade) 0        
-                                                                                
-Unit Upkeep       Shield   Shield   Gold     Gold     Gold(x2) Shield   Gold(x2) Gold(x2) Shield   
-Free Units        2\2      2\2      2\2      3\3      2\2      1\1      1\1      2\2      3\3      
-Mil Unhappiness   0        0        0       -1       -1       -1       -2       -1       -1       
-Martial Law      +1        0       +1       +1        0        0        0        0       +2       
-                                                                                
-Civil war %       99       90       80       70       60       40       30       20       50       
-Empire Size       0        /12      /10      /14      /20      /16      /32      /24      /28      
-Special          +NoUpkeep +Veteran +75%Gold +50%Gold +Tithes  +Bribe  +Brib/Sub +Subvert +NoUpkeep
-                 -50%Lux  (Land/Mil)(Palace) (Palace) -50%Sci  -Revolt -Rev/Sen  -Senate  (Cost=1) 
-                 -NoTaxes +Content                    +Fanatic         +Partisan +Luxury  +Partisan 
-                                                                                
-Base Corruption   0%       30%      20%      10%      15%      25%      5%       0%       30%      
-Corrup by Dist *  1        1        1        1        1        1        1        0        0        
-Base Waste        30%      0%       10%      20%      15%      5%       25%      30%      0%       
-Waste by Dist **  1        1        1        1        1        1        1        0        0        
-
-* Corruption by Distance is doubled until The Corporation is researched
+
+==================== ========== ========== ======== ========= ======== ========= ========= ========= ==========
+Item                 Anarchy    Tribal     Despot   Monarchy  Funda    Republic  Democracy Fed       Communism
+==================== ========== ========== ======== ========= ======== ========= ========= ========= ==========
+Tax/Lux/Sci Rate     0          60         60       70        70       80        90        90        80
+Output per tile (#)  -1(if>2)   -1(if>2)   -1(if>2) 0         0        +1Trade   +1Trade   0         0
+Celebration Bonus    (no -1)    (no -1)    (no -1)  (+1Trade) 0        (+1Trade) (+1Trade) (+1Trade) 0
```

Comment:
Can we have a space between the value and unit (`+1 Trade` here, `-50% Sci` few lines down)? I think this improves readability